File: Language_Coder/src/loop/Code633.java
package loop;

import java.util.Scanner;

public class Code633 {

	public static void main(String[] args) {

		Scanner scan = new Scanner(System.in);

		while (true) {
			System.out.println("1. Korea\r\n" + "2. USA\r\n" + "3. Japan\r\n" + "4. China");
			System.out.print("number? ");
			int command = scan.nextInt();
			System.out.println();

			switch (command) {
			case 1:
				System.out.println("Seoul");
				break;
			case 2:
				System.out.println("Washington");
				break;
			case 3:
				System.out.println("Tokyo");
				break;
			case 4:
				System.out.println("Beijing");
				break;
			default:
				System.out.println("none");
				return;
			}

			System.out.println();
		}
	}

}

File: Language_Coder_Cpp/Code633/Code633.cpp
// Code633.cpp

#include <iostream>
using namespace std;

int main()
{
	while (true) {
		cout << "1. Korea" << endl;
		cout << "2. USA" << endl;
		cout << "3. Japan" << endl;
		cout << "4. China" << endl;
		cout << "number? ";

		int num;
		cin >> num;

		cout << endl;

		switch (num) {
		case 1 :
			cout << "Seoul" << endl;
			break;
		case 2:
			cout << "Washington" << endl;
			break;
		case 3:
			cout << "Tokyo" << endl;
			break;
		case 4:
			cout << "Beijing" << endl;
			break;
		default :
			goto End;
		}

		cout << endl;
	}

End: cout << "none" << endl;
}